team_picker/views/show.ejs

```diff
@@ -0,0 +1,82 @@
+<%- include('./partials/header') %>
+
+<img class="card-img-top logo ml-auto mr-auto" src="<%=cohort.logoUrl%>" alt="Card image cap">
+<h1><%=cohort.name%> </h1>
+<ul>
+  <% for (let member of cohort.members.split(",")) { %>
+    <li class="card-text"><%=member%></li>
+  <% } %>
+</ul>
+<div class="d-flex">
+  <a href="/teams/<%= cohort.id %>/edit" class="btn btn-outline-success btn-sm mr-2">Edit</a>
+  <form action="/teams/<%= cohort.id %>" METHOD="POST">
+    <input type="hidden" name="_method" value="DELETE">
+    <input type="submit" value="Delete" class="btn btn-sm btn-outline-danger">
+  </form>
+</div>
+<br>
+<div>
+  <h4>Team Picker</h3>
+  <form action="/teams" method="POST">
+    <input type="hidden" name="id" value="<%= cohort.id %>">
+    <div class="form-row">
+      <div class="form-group col-md-4">
+        <select class="form-control" name="pickerMethod">
+          <option selected>Choose...</option>
+          <option value="totalTeams">How many teams?</option>
+          <option value="membersPerTeam">How many members per team?</option>
+        </select>
+      </div>
+      <div class="col-md-2">
+        <input type="text" class="form-control" name="quantity" placeholder="Quantity">
+      </div>
+    </div>
+    <button type="submit" class="btn btn-outline-primary">Assign teams</button>
+  </form>
+</div>
+
+<% if (pickerMethod) { %>
+  <% const randomizedMembersArray = cohort.members.split(",").sort(function(a, b){return 0.5 - Math.random()}) %> 
+  <div class="row">
+    <% if (pickerMethod == "totalTeams") { %>
+      <% membersPerTeam = Math.round(randomizedMembersArray.length / quantity) %>
+      <% for (let i = 1; i <= quantity; i++ ) { %>
+        <div class="col-sm-3">
+          <div class="card">
+            <div class="card-body">
+              <h5 class="card-title">Team <%= i %></h5>
+              <ol class="card-text">
+                <% for (let member of randomizedMembersArray.splice(0,membersPerTeam)) { %>
+                  <li class="card-text"><%=member%></li>
+                <% } %>
+                <% if (randomizedMembersArray.length > 0 && randomizedMembersArray.length < membersPerTeam) { %>
```

Comment:
Coupled with the comment about moving the logic out of the view and into the routes, it's quite difficult to reason about what's going on here when you're mixing display and business logic, which I think will be greatly simplified once you move that out of the views. Also, you'll be able to eliminate a great deal of the duplication you have in the setup of the cards in the two sides of the [`if`](https://github.com/rosadiaz/homeworks/pull/6/files#diff-2e283d0a74b1a86486b7f3457d4d79f3R41)